app/assets/stylesheets/rat_adoptions.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the rat_adoptions controller here.
```

Comment:
How would you feel about removing this file and renaming `app/assets/stylesheets/adopt.css.scss` to `app/assets/stylesheets/rat_adoptions.scss`?